lib/cc/config.rb

```diff
@@ -20,9 +20,9 @@ class Config
     def self.load
       config = Default.new
       if File.exist?(JSON::DEFAULT_PATH)
-        config = config.merge(JSON.new)
-      elsif File.exist?(YAML::DEFAULT_PATH)
-        config = config.merge(YAML.new)
+        config = config.merge(JSON.load)
```

Comment:
It's a little confusing that we're using `JSON` as the class name. Would it make sense to rename this to `JSONAdapter`?

Reply:
That's coming! I'm trying to make smaller changes so the diffs don't get too crazy with just name changes. I'm happy to keep pushing on this branch in commits if you'd prefer to review that way.